V2MOM/v2.0.0.md

```diff
@@ -0,0 +1,39 @@
+# V2MOM v2 Draft
+
+## Vision
+
+Enable the project and core teams to achieve their goals through the use of technological tools and resources.
+
+## Values
+
+* Quality: Lead by example; build quality projects
+* Accessibility: Ensure that projects we support are accessible
+* Transparency: Develop in the open
```

Comment:
Thought about this a bit more and felt like noting the exclusion of secrets / sensitive data was noise, but let me know if you still feel strongly about it.